VIRTUALJOINT_1: 0
VIRTUALJOINT_2: 0
VIRTUALJOINT_3: 0
VIRTUALJOINT_4: 0
VIRTUALJOINT_5: 0
VIRTUALJOINT_6: 0
hip_yaw_1: -1.161091630963811
hip_pitch_1: 1.671818763185724
knee_pitch_1: -2.325504140167387
ankle_pitch_1: 0.1792223369559239
ankle_yaw_1: -0.1271459610177637
j_wheel_1: -2.858581316272381e-08
hip_yaw_2: -0.2234847283267875
hip_pitch_2: -2.058737273963697
knee_pitch_2: 1.601621777682543
ankle_pitch_2: -0.8237116124333272
ankle_yaw_2: 0.3983247851232499
j_wheel_2: 5.47740077999097e-09
hip_yaw_3: -0.904478286941251
hip_pitch_3: 1.461833793500144
knee_pitch_3: -0.8792080928474049
ankle_pitch_3: 0.3814818097906991
ankle_yaw_3: 0.0008682531488266389
j_wheel_3: -4.170726152158972e-09
hip_yaw_4: 0.4041266117928506
hip_pitch_4: 0.01810308089422936
knee_pitch_4: -0.3753199562648274
ankle_pitch_4: 0.291156442334598
ankle_yaw_4: -0.2610968002196835
j_wheel_4: 1.813960291576824e-09
torso_yaw: 5.780825638053133
neck_velodyne: -65.52025913935138
neck_yaw: -65.51981529621902
neck_pitch: -65.92097106916044
j_arm1_1: 3.158023981662278
j_arm1_2: 4.32970413640436
j_arm1_3: -0.687804620788208
j_arm1_4: -1.436872354147631
j_arm1_5: -0.6058362998817753
j_arm1_6: -0.3096313649317165
j_arm1_7: -0.04958104607093577
j_arm2_1: 4.727118056322357
j_arm2_2: -9.451832043765632
j_arm2_3: 1.235552236725397
j_arm2_4: -0.9545216925896345
j_arm2_5: 0.6546875543506797
j_arm2_6: 0.2667281546181865
j_arm2_7: -0.1475264642372433